frontend/src/modules/Dashboard/Types/index.ts

```diff
@@ -0,0 +1,6 @@
+export interface CourseInfo {
```

Comment:
I think in other components this is not in a file named `index.ts`, rather that's used to re-export the content of the files